cerberus/cerberus.py

```diff
@@ -298,6 +316,9 @@ def _validate_definition(self, definition, field, value):
             if validator:
                 validator(definition[rule], field, value)
 
+        if 'rename' in definition:
```

Comment:
i mean the possibility to override this option via a rule, that can be placed in a schema-defintion. since `allow_unknown` (as `Validator`-property) is inherited by derived validators, there is also the constraint `allow_unknown` in order to override this. please see the documentation about it.